GVRf/Framework/src/org/gearvrf/GVRPicker.java

```diff
@@ -362,22 +362,22 @@ public GVRSceneObject getHitObject() {
          *         {@link GVRPickedObject picked object's} hit data.
          */
         public float[] getHitLocation() {
-            return Arrays.copyOf(hitLocation, hitLocation.length);
+            return Arrays.copyOf(mHitLocation, mHitLocation.length);
         }
 
         /** The x coordinate of the hit location */
         public float getHitX() {
-            return hitLocation[0];
+            return mHitLocation[0];
         }
 
-        /** The x coordinate of the hit location */
+        /** The y coordinate of the hit location */
```

Comment:
haha, good catch.  not sure how we missed these two copy-n-paste typos.  thanks.